src/main/java/thaumicboots/api/IComet.java

```diff
@@ -0,0 +1,4 @@
+package thaumicboots.api;
+
+public interface IComet {
+}
```

Comment:
what's the point of having empty interfaces?

Reply:
It's for effects that are handled in a different part of the mod, if this gets hit an effect happens, specifically the ice and meteor effect

Reply:
done